docs/docs-beta/docs/guides/ingesting-data.md

```diff
@@ -2,54 +2,56 @@
 title: Ingesting data with Dagster
 description: Learn how to orchestrate data ingestion with Dagster
 sidebar_position: 10
-sidebar_label: Ingest data with embedded-elt
+sidebar_label: Ingest data
 ---
 
-Dagster is often used to orchestrate the ingestion of data into a data warehouse or data lake, where it can be queried and transformed. Dagster integrates with several tools that are purpose-built for data ingestion, and it also enables writing custom code for ingesting data.
-
-This guide covers how to use Dagster for data ingestion.
+import { Card, CardGroup } from '@site/src/components/Cards';
 
 :::note
 This guide focuses on batch data ingestion, as streaming data ingestion doesn't typically rely on an orchestrator to kick off or coordinate computations. However, streaming data assets can still be represented in Dagster for lineage purposes.
 :::
 
+Dagster is often used to orchestrate the ingestion of data into a data warehouse or data lake, where it can be queried and transformed. To ingest data with Dagster, you can use pre-built connectors or write your own custom code.
+
 <details>
-  <summary>Prerequisites</summary>
+<summary>Prerequisites</summary>
+
+To follow this guide, you'll need:
+
 - Familiarity with [Assets](/concepts/assets)
 </details>
 
-## How Dagster helps with data ingestion
+## How Dagster supports data ingestion
 
 As a data orchestrator, Dagster helps with data ingestion as it can:
 
 - **Automatically kick off computations that ingest data**, thus removing the need for manual intervention
-- **Coordinate data ingestion with downstream data transformation,** such as rebuilding a set of dbt models after the upstream data they depend on is updated
+- **Coordinate data ingestion with downstream data transformation,** such as rebuilding a set of dbt models after upstream data is updated
 - **Represent ingested data assets in an asset graph**, which enables understanding what ingested data exists, how ingested data is used, and where data is ingested from
 
 ## Orchestrating data ingestion tools
 
-Dagster integrates with a variety of data ingestion tools, enabling you to sync diverse data sources into your data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
+Dagster integrates with a number of data ingestion tools, enabling you to sync diverse data sources into data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
 
 - Represent ingested tables as assets within the Dagster asset graph
 - Trigger asset materializations that automatically invoke these tools to initiate data syncs
 
 Dagster currently integrates with the following data ingestion tools:
 
-- [Airbyte](/todo)
-- [dlt](/todo)
-- [Fivetran](/todo)
-- [Sling](/todo)
+- Airbyte
```

Comment:
I do think we'll want these to be links at some point, but not a blocker for now.

Reply:
Thanks for catching this!

Reply:
It looks like Yuhan merged in the integration pages, so I can add the links!